plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java

```diff
@@ -890,6 +915,11 @@ public static void register() {
         // @mechanism CuboidTag.add_member
         // @description
         // Returns a modified copy of this cuboid, with the input cuboid(s) added at the end.
+        // @example
+        // # Adds "my_second_cuboid" as a member of "my_cuboid".
```

Comment:
Not quite. It creates a *new* cuboid, named `my_third_cuboid`, which contains both `my_cuboid` and `my_second_cuboid` as members